config/crd/bases/delivery.ocm.software_componentversions.yaml

```diff
@@ -93,10 +92,12 @@ spec:
                   or destination, but it is still allowed to do so. https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account
                 type: string
               suspend:
-                description: Suspend stops all operations on this component version
-                  object.
+                description: Suspend can be used to temporarily pause the reconciliation
+                  of the ComponentVersion resource.
                 type: boolean
               verify:
+                description: Verify specifies a list signatures that should be verified
+                  before the ComponentVersion is marked Ready.
```

Comment:
Verify specifies a list of signatures that should be used for verification before the ComponentVersion is marked a Success.

Even a failed Component Version is marked ready